patches/tModLoader/Terraria.ModLoader/Logging.cs

```diff
@@ -22,7 +22,8 @@ namespace Terraria.ModLoader
 {
 	public static class Logging
 	{
-		public static readonly string LogDir = Path.Combine(Program.SavePath, "Logs");
+		public static readonly string LogDir = Path.Combine(Program.SavePath, "Logs");
+		public static readonly string LogArchiveDir = Path.Combine(Path.Combine(Program.SavePath, "Logs"), "Old");
```

Comment:
Reuse LogDir here?

Reply:
10pm me thought that could cause shenanigans from LogDir not being assigned. If not ill change it

Reply:
10pm me thought that could cause shenanigans from LogDir not being assigned. If not ill change it